strongbox-metadata-maven-api/src/test/java/org/carlspring/strongbox/storage/metadata/MetadataHelperTest.java

```diff
@@ -84,13 +85,22 @@ public static void down()
 
     private static void deleteTestResources()
     {
-        Path dirPath = Paths.get(pluginXmlFilePath).getParent();
+
+        Path dirPath = Paths.get(pluginXmlFilePath).getParent().getParent().getParent();
```

Comment:
Is this really the right way to be doing this?

Reply:
Since I am creating the files on the fly, I have to delete the file and folders from parent.

Reply:
I couldn't find another way pf doing it